plugins/micromega/simplex.ml

```diff
@@ -516,9 +534,7 @@ let cut env rmin sol vm (rst : Restricted.t) tbl (x, v) =
         Vect.null r
     in
     let lcut =
-      List.map
-        (fun cv -> Vect.normalise (cut_vector cv))
-        [cut_coeff1; cut_coeff2]
+      List.map (fun cv -> Vect.normalise (cut_vector cv)) [cut_coeff1]
```

Comment:
Using `List.map` on a list with exactly one element seems a bit overkill.

Reply:
Quite right. There used to be two elements. I'll fix that.